coconut/control/control.go

```diff
@@ -278,15 +279,30 @@ func CreateEnvironment(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.C
 	payloadData := new(ConfigurationPayload)
 	var wfPath string
 	if cmd.Flags().Changed("configuration") && len(configPayload) > 0 {
-		configPayloadDoc, err := os.ReadFile(configPayload)
-		if err != nil {
-			err = errors.New("cannot read configuration payload file")
-			return err
-		}
-		err = json.Unmarshal(configPayloadDoc, &payloadData)
-		if err != nil {
-			err = errors.New("cannot unmarshal configuration payload")
-			return err
+		if strings.HasSuffix(strings.ToLower(configPayload), ".json") {
+			configPayloadDoc, err := os.ReadFile(configPayload)
+			err = json.Unmarshal(configPayloadDoc, &payloadData)
+			if err != nil {
+				err = errors.New("cannot unmarshal local configuration payload")
+				return err
+			}
+		} else {
+			configPayloadSplit := strings.Split(configPayload, "/")
+			if len(configPayloadSplit) == 0 {
+				err = errors.New("configuration payload file is not valid")
+				return err
+			}
+			configPayloadUri := configPayloadSplit[len(configPayloadSplit)-1]
+			configPayloadDoc, err := apricot.Instance().GetRuntimeEntry("COG-v1", configPayloadUri)
```

Comment:
Minor nitpick: please put `"COG-v1"` in a `const HLCONFIG_RUNTIME_COMPONENT_PREFIX` on top of the file, or similar.